reviews/review-03+2020-11-30.md

```diff
@@ -0,0 +1,169 @@
+﻿1. What do you think is good about this manuscript?
+
+a. This is a solid presentation of often overlooked skills for designing not
+just useful, but successful software for scientific research. It is all too easy
+to develop software that "works" but simply working is a lower bar that most
+researchers realize. Software is essential to research---especially in the
+natural sciences---but it is a part of the research workflow that can easily
+evade scrutiny (few peer reviews include code review). This book presents a
+recipe for skills (computational and collaborative) that can help research
+software developers deliver reproducible science and build confidence that their
+work can be sustained even after their initial contributions.
+
+b. Notes/callouts and glossary are very helpful for the reader. Code
+instructions are clear, thorough, and nicely commented.
+
+2. What reservations do you have about this manuscript?
+
+a. Chapters on their own a solid. I think there are times where I felt myself
+wanting a bit more "glue" or context. For example, as currently presented,
+learning objectives are separated from chapters. I want to know at the beginning
```

Comment:
This is a good point. We include our key points in an appendix and at the end of each chapter, but our learning objectives are only included in an appendix. I'd support including the learning objectives at the beginning of each chapter too (immediately before or after the Terry Pratchett quote).

Reply:
I'm :-1: on this:

1.  Now that we have better opening sentences for chapters and sections, readers will know what the road ahead looks like.

2. It adds length to the print version (which is already quite long).

3. They're most useful to instructors teaching courses, for whom the consolidated version at the back is most useful.

4. It's December 1 and we need to ship this, so I'm :-1: on non-essential changes in general.